### ISSUE TITLE
New changes for sensor

### ISSUE THREAD

**Issue by gkeishin** [openbmc/openbmc-test-automation#1438]
Changes from @anoo1  to address properties for /org/open_power/sensor to /xyz/ path.

Impacts test. Need to identify which are test cases are impacted

**Comment by anoo1** [comment]
- BootProgress sensor -> /xyz/openbmc_project/state/host0/attr/BootProgress

- OperatingSystemsStatus sensor -> /xyz/openbmc_project/state/host0/attr/OperatingSystemState

- BootCount sensor -> /xyz/openbmc_project/state/host0/attr/AttemptsLeft

- TurboAllowed sensor -> /xyz/openbmc_project/control/host0/turbo_allowed/attr/TurboAllowed

- PowerSupplyRedundancy sensor -> /xyz/openbmc_project/control/power_supply_redundancy/attr/PowerSupplyRedundancyEnabled

- PoweSupplyDeratingFactor sensor -> (Under review) /xyz/openbmc_project/control/power_supply/attr/DeratingFactor

**Comment by ssombar** [comment]
I am thinking of doing these one at a time, each with with a separate issue and commit.

**Comment by gkeishin** [comment]
sounds good to me @ssombar  lets do separate commits so that we dont break things ..

**Comment by ssombar** [comment]
We can't commit our changes yet.  Adriana advises that she's waiting on some reviews to get the derating factor path approved.

**Comment by gkeishin** [comment]
I think it would be wise to wait until the path URI is confirmed and then test? However, if we are confident that @anoo1 changes URI won't change, then yeah we can use the test build and check in the code @ssombar

**Comment by ssombar** [comment]
https://gerrit.openbmc-project.xyz/#/c/openbmc/openbmc-test-automation/+/12168/

**Comment by ssombar** [comment]
issue 1455 addresses changes for PowerSupplyRedundancy sensor https://github.com/openbmc/openbmc-test-automation/issues/1455

**Comment by ssombar** [comment]
OK to close, I think.
